opencti-platform/opencti-graphql/src/schema/fieldDataAdapter.ts

```diff
@@ -20,6 +20,9 @@ import { MULTIPLE_STIX_CYBER_OBSERVABLE_RELATIONSHIPS_INPUTS } from './stixCyber
 import type { StixArtifact, StixFile, StixX509Certificate } from '../types/stix-sco';
 import type { HashInput } from '../generated/graphql';
 
+export const SENSITIVE_HASHES = ['SSDEEP', 'SDHASH'];
+export const FUZZY_HASH_ALGORITHMS = ['SSDEEP', 'SDHASH', 'TLSH', 'LZJD'];
```

Comment:
are the two arrays related?

export const FUZZY_HASH_ALGORITHMS = [...SENSITIVE_HASHES, 'TLSH', 'LZJD'];